.github/workflows/main.yml

```diff
@@ -40,6 +40,7 @@ jobs:
             export TEST="E2E";
           fi
           docker run \
+            --ulimit stack=-1 \
```

Comment:
This is to workaround the following error in R test which didn't occur on Bionic.
```
        stdout, stderr = result.communicate()
        if result.returncode != 0:
            raise Exception(
>               f"Bad exit code ({result.returncode}), "
                f"stderr:\n{stderr.decode('utf-8')}")
E           Exception: Bad exit code (1), stderr:
E           Error: C stack usage  7974932 is too close to the limit
E           Execution halted
...
test_e2e[sklearn_SVR - r_lang - train_model_regression]
```